Tests/scripts/hook_validations/integration.py

```diff
@@ -84,7 +85,7 @@ def is_default_arguments(self):
                         or (command_name == 'domain' and arg_name == 'domain')
                         or (command_name == 'url' and arg_name == 'url')
                         or (command_name == 'ip' and arg_name == 'ip')):
-                    if arg.get('default') is False or arg.get('required') is True:
```

Comment:
@Itay4 
does the argument needs to be required or we don't care about the status of it??

Reply:
doesn't have to be required, just makes sense it will be.. cant think of a case it shouldnt be, but dont see a reason to force it in the validation